votey/slack.py

```diff
@@ -219,21 +225,26 @@ def get_command_from_req(
             'Try again with `/votey "question" "option 1" "option 2"`',
         )
         return None, False
-    if len(split) > 7 or (not is_anonymous(split) and len(split) > 6):
+    if len(split) > 12 or (not is_anonymous(split) and len(split) > 11):
         send_ephemeral_message(
             workspace,
             request.get('channel_id', ''),
             request.get('user_id', ''),
-            'Sorry - Votey only supports 5 options at the moment, ' \
-            'due to limitations in the Slack API',
+            'Sorry - Votey only supports 10 options at the moment.',
         )
         return None, False
     if is_anonymous(split):
         return split[0:-1], True
```

Comment:
What if `-anon` is in the middle of the command?

Looks like we'll need a little extra logic here unfortunately, since we support multiple anon options, I dont think we can simply call `split.remove()` or something like that